kapprover/kapprover.go

```diff
@@ -73,10 +122,12 @@ func tryApprove(filters inspectors.Inspectors, deniers inspectors.Inspectors, wa
 		for _, filter := range filters {
 			message, err := filter.Inspector.Inspect(client, request)
 			if err != nil {
+				requestsError.WithLabelValues("filterError").Inc()
 				return err
 			}
 			if message != "" {
 				log.Infof("Skipping %q from %q: %s", request.Name, request.Spec.Username, message)
+				requestsFiltered.WithLabelValues().Inc()
```

Comment:
Throw in the insepctor.Name as a label value?

Reply:
Done